example/cr/jsonschema-crd.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.schemaregistry.enabled }}
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```

Comment:
This is a CRD, it shouldn't be in the example. The individual services definitely won't be deploying this, who is deploying it?

Reply:
fair enough. I'll remove it